docs/content/getting-started/create-new-project.mdx

```diff
@@ -45,9 +47,7 @@ Here's a breakdown of the files and directories that are generated:
   <tbody>
     <tr>
       <td>my_dagster_project/</td>
-      <td>
-        A Python package that contains code for your new Dagster repository.
-      </td>
+      <td>A Python package that contains code for your new Dagster code.</td>
```

Comment:
Yo dawg, I heard you like code, so I got some code for your code!

lol. Perhaps this should be something like `A Python package that contains code for your new Dagster project`?

Reply:
"A Python package that contains your new Dagster code"?